### PR TITLE
Update to crest 3.0.1

### PR DESCRIPTION
**Pull request by maxscheurer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by maxscheurer** [comment]
If anyone knows how to fix the "undefined reference to..." error, that would be great 😁

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,14 +35,14 @@ requirements:
     - libblas
     - liblapack
```
Comment (awvwgk) [inline recipe/meta.yaml:36]:
```suggestion
    - liblapack
    - tblite
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -34,14 +35,14 @@ requirements:
...
   run:
-    - xtb
+    - tblite
```
Comment (awvwgk) [inline recipe/meta.yaml:38]:
```suggestion
```

**Review by awvwgk — CHANGES_REQUESTED** [review]
Also, please remove the static library from the package as those should not be included in the distribution for cf.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -18,8 +18,9 @@ build:
...
     - cmake --build build_dir
     - cmake --install build_dir
-    - mkdir -p $PREFIX/share/man/man1
-    - asciidoctor --attribute=release-version={{ version }} --backend=manpage --out-file=$PREFIX/share/man/man1/crest.1 docs/man/crest.adoc
+    - cd build_dir && make test
+    # - mkdir -p $PREFIX/share/man/man1
+    # - asciidoctor --attribute=release-version={{ version }} --backend=manpage --out-file=$PREFIX/share/man/man1/crest.1 docs/man/crest.adoc
 
 requirements:
   build:
```
Comment (awvwgk) [inline recipe/meta.yaml:26]:
```suggestion
  build:
    - {{ stdlib('c') }}
```

**Comment by maxscheurer** [comment]
@conda-forge-admin, please rerender.

**Comment by maxscheurer** [comment]
Superseded by #13

**Pull request closed without merging**